### ISSUE TITLE
This browser won’t be supported starting Sept. 1, 2021.

### ISSUE THREAD

**Issue by cmoulliard** [getferdi/ferdi#1552]
## Description

Some of my slacks services reports the following message at the bottom part of the screen that I wanted to share with you. Do you think that Ferdi will continue to be able to work with the Slack servers ?

`This browser won’t be supported starting Sept. 1, 2021. Update your browser or get the latest app to keep using Slack. Learn more`

![Screenshot 2021-06-23 at 13 46 33](https://user-images.githubusercontent.com/463790/123091300-795fc800-d429-11eb-8cda-b0ee2e15ce80.png)

**Comment by vraravam** [comment]
@cmoulliard - please upgrade to the latest nightlies found [here](http://github.com/vraravam/nightlies/releases) - i am using slack on Ferdi (nightly) - and don't see this issue.

Also a request: when logging tickets, it would be a good idea to post which version of Ferdi you are using , which OS, etc. There's a template for such items in GH.

**Comment by cmoulliard** [comment]
Why are you closing the ticket ?

Here is the info requested
```
Ferdi : Version 5.6.0-beta.2 (5.6.0-beta.2.2329)
MacOs: BigSur 11.4
```

**Comment by vraravam** [comment]
@cmoulliard - we are free to reopen the ticket. As I mentioned above, this is not an issue with the latest code (not yet released). Please upgrade to the nightliy version (which is quite stable) and you can get the latest version of Chromium that Ferdi is built with.

**Comment by vraravam** [comment]
With one of the PRs that got merged in over the past week or so, the user will also be able to specify their own string for the `user-agent` - which is how Slack determines whether the browser is supported or not. Feel free to explore that option as well for your usecase.

**Comment by cmoulliard** [comment]
Many thanks for the clarification. If this ticket will be fixed with the upcoming 5.6 release, then you could perhaps add here the link to the PR or milestone to keep us informed  ;-)

**Comment by vraravam** [comment]
[This is the PR](https://github.com/getferdi/ferdi/pull/1547) that merged the latest version of electron 13.1.4 - of course, your issue might have been fixed in an earlier version as well. Please try the latest nightly and close this issue once you can confirm this.

**Comment by tcoulter** [comment]
For what it's worth: I just updated to the 5.6.0-nightly.66 and don't immediately see the notification. Thanks!
